lando/k8s/jobmanager.py

```diff
@@ -24,6 +24,11 @@ class JobStepTypes(object):
     RECORD_OUTPUT_PROJECT = "record_output_project"
 
 
+class WorkflowTypes(object):
```

Comment:
Can we centralize these definitions rather than duplicating them? Ideally we'd define the types (and their special logic) in one place. [lando.worker.cwlworkflow](https://github.com/Duke-GCB/lando/blob/8bfc654ab08a1391bf18dc83bb40773e66e13bd2/lando/worker/cwlworkflow.py#L83-L86) already has quite a bit of this logic that could be refactored into a single piece of code that both share.

Reply:
Created issue #197